content/includes/header.html

```diff
@@ -34,6 +34,9 @@
 <meta content="{{ tag }}" property="article:tag">
 {% endfor %}
 {% endif %}
+{% if title == 'Education Benefits Application Process' or title == 'Education Benefits Eligibility' %}
+<link rel="import" href="/education/wizard/index.html">
```

Comment:
Did you try to do this with the `include` directive from Metalsmith (or our markdown processor, not sure exactly where it's from)? We use it for a similar reusable component in the content pages, the sticky action box:

`{% include "content/includes/sticky-action-box.html" %}` (in 6-info-page.html).